server/pulp/server/async/manage_workers.py

```diff
@@ -17,11 +17,12 @@
 After=network.target
 
 [Service]
+UMask=0002
```

Comment:
@bmbouter @jlsherrill @daviddavis @mikedep333 @rohanpm @goosemania can people provide their input on are there any security implications that groups will have write access on content files?

```
$ ll /var/lib/pulp/content/units/rpm/0d/99557451f5310e404ed9d984706d9d9783368aab1056655a3e5c7cf69354e7/cheetah-1.25.3-5.noarch.rpm 
-rw-rw--r--. 1 apache pulp 1880 Oct 21 15:30 /var/lib/pulp/content/units/rpm/0d/99557451f5310e404ed9d984706d9d9783368aab1056655a3e5c7cf69354e7/cheetah-1.25.3-5.noarch.rpm
```

Reply:
also, pulp3 creates artifacts with the following permissions:
```
$ ll /var/lib/pulp/artifact/c8/16d043c412cea80218394df3a3d61d05f531ee2322cd184a383c5a31611283 
-rw-------. 1 vagrant pulp 1024 Oct 21 17:41 /var/lib/pulp/artifact/c8/16d043c412cea80218394df3a3d61d05f531ee2322cd184a383c5a31611283
```

Reply:
i'm assuming in this case your pulp_workers were running as the vagrant user?

Reply:
In the current deployment with katello the only other user in the pulp group is apache, and since apache is the owner, i have no concerns

Reply:
I expect the group ownership to be `pulp` and anyone in that group has received access. I also have no concerns with the `pulp` group having full write access.

Reply:
Likewise.